apps/frontend/src/constants/gasLimits.ts

```diff
@@ -24,4 +24,5 @@ export const GAS_LIMIT = {
   BORROW: 1_500_000,
   REPAY_LOAN: 950_000,
   WITHDRAW_LOAN_COLLATERAL: 150_000,
+  PROPOSAL_ACTION: 250_000,
```

Comment:
make different actions for queue and execute, for queing 250k will be enough (maybe even less will work), but for execution we may need full 6M